Docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  Development:
+    env_file:
+        - .env
+    build: . 
+    
+    .volumes: 
+        - type: bind
+          source: /todo_app
+          target: /app
+
```

Comment:
You need to publish a port if you want to actually visit it in your browser. E.g:

```
    ports:
      - "5000:5000"
```

(Equivalent to the docker command line's `-p` option)